app/src/main/java/com/demoapp/dagger/AppComponentProvider.kt

```diff
@@ -0,0 +1,7 @@
+package com.demoapp.dagger
+
+interface AppComponentProvider {
+
+   fun  getAppComponent( ) : AppComponent
+
+}
```

Comment:
Ensure enplane for all files

Reply:
Raghu do you mean need to all activities or all the files requiring dependencies?

Reply:
nope. end line

Reply:
removed end line

Reply:
Not to remove end line. we need enplane. Github should not show and icon like 🚫